packages/components/src/dropdown/style.scss

```
@@ -31,9 +31,9 @@
 		margin-top: 0;
 		border-top: $border-width solid $gray-400;
 		padding: $grid-unit-10;
-
-		.is-alternate & {
-			border-color: $gray-900;
-		}
 	}
 }
+
+.is-alternate .components-menu-group + .components-menu-group {
+	border-color: $gray-900;
+}
```

Since `.is-alternate` is a modifier of `.components-dropdown__content` does it make sense to move the rule there?

```suggestion
	}

	&.is-alternate .components-menu-group + .components-menu-group {
		border-color: $gray-900;
	}
}
```